site/src/testHelpers/storybook.tsx

```diff
@@ -121,6 +122,30 @@ export const withAuthProvider = (Story: FC, { parameters }: StoryContext) => {
 	);
 };
 
+export const withProvisioners = (Story: FC, { parameters }: StoryContext) => {
+	if (!parameters.organization_id) {
+		throw new Error(
+			"You forgot to add `parameters.organization_id` to your story",
+		);
+	}
+	if (!parameters.provisioners) {
+		throw new Error(
+			"You forgot to add `parameters.provisioners` to your story",
+		);
+	}
+	if (!parameters.tags) {
+		throw new Error("You forgot to add `parameters.tags` to your story");
```

Comment:
optional, but might make things more concise

```suggestion
			"Add `parameters.organization_id` to your story",
		);
	}
	if (!parameters.provisioners) {
		throw new Error(
			"Add `parameters.provisioners` to your story",
		);
	}
	if (!parameters.tags) {
		throw new Error("Add `parameters.tags` to your story");
```